sklearn/tree/_tree.pxd

```diff
@@ -27,37 +27,33 @@ cdef class Criterion:
     # impurity of a split on that node. It also computes the output statistics
     # such as the mean in regression and class probabilities in classification.
 
-    # Internal structures
```

Comment:
It is bad practice, but since this is still a work in progress I have been a little lax with keeping documentation up to date, as I have been changing things fairly frequently.